ui/components/app/collectibles-tab/collectibles-tab.js

```diff
@@ -14,24 +16,61 @@ import {
   FONT_WEIGHT,
 } from '../../../helpers/constants/design-system';
 import { useI18nContext } from '../../../hooks/useI18nContext';
+import {
+  getCollectibles,
+  getCollectibleContracts,
+  getCollectiblesDetectionNoticeDismissed,
+} from '../../../ducks/metamask/metamask';
+import { getIsMainnet, getUseCollectibleDetection } from '../../../selectors';
+import { EXPERIMENTAL_ROUTE } from '../../../helpers/constants/routes';
+import { detectCollectibles } from '../../../store/actions';
 
 export default function CollectiblesTab({ onAddNFT }) {
-  const collectibles = [];
-  const newNFTsDetected = false;
+  const collectibles = useSelector(getCollectibles);
+  const collectibleContracts = useSelector(getCollectibleContracts);
+  const useCollectibleDetection = useSelector(getUseCollectibleDetection);
+  const isMainnet = useSelector(getIsMainnet);
+  const collectibleDetectionNoticeDismissed = useSelector(
+    getCollectiblesDetectionNoticeDismissed,
+  );
+  const history = useHistory();
   const t = useI18nContext();
+  const dispatch = useDispatch();
+
+  const collections = {};
+  collectibles.forEach((collectible) => {
+    if (collections[collectible.address]) {
+      collections[collectible.address].collectibles.push(collectible);
+    } else {
+      const collectionContract = collectibleContracts.find(
+        ({ address }) => address === collectible.address,
+      );
+      collections[collectible.address] = {
+        collectionName: collectionContract?.name || collectible.name,
+        collectionImage:
+          collectionContract?.logo || collectible.collectionImage,
+        collectibles: [collectible],
+      };
+    }
+  });
 
   return (
     <div className="collectibles-tab">
       {collectibles.length > 0 ? (
         <CollectiblesItems
+          collections={collections}
           onAddNFT={onAddNFT}
-          onRefreshList={() => {
-            console.log('refreshing collectibles');
-          }}
+          useCollectibleDetection={useCollectibleDetection}
+          onRefreshList={() => dispatch(detectCollectibles())}
+          onEnableAutoDetect={() => history.push(EXPERIMENTAL_ROUTE)}
         />
       ) : (
         <Box padding={[6, 12, 6, 12]}>
-          {newNFTsDetected ? <NewCollectiblesNotice /> : null}
+          {isMainnet &&
```

Comment:
I'm noticing that if this is dismissed, we continue to only show "Add NFT" as an option under "Don't see your NFT?". Should we have it say "Enable Autodetect or Import NFTs" just like what is shown if you do have NFTs (but again, only if this notice has been dismissed)?

Reply:
hmmm possibly. @rachelcope What do you think? Should we show the "Enable Autodetect" link in the case that Elliot is describing here?